### PR TITLE
chore: replace cluster status and explorer configuration with about handler

### PR DESCRIPTION
**Pull request by hugoArregui**
The about endpoint will return:

- Comms version
- Service health (by querying lambda if v2, or querying lambda + archipelago status discovery if v3)

### CODE REVIEW

File: src/controllers/handlers/about-handler.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  }
+  lambdas: {
+    healthy: boolean
+  }
+}
+
+// handlers arguments only type what they need, to make unit testing easier
+export async function aboutHandler(
+  context: Pick<
+    HandlerContextWithPath<'serviceDiscovery' | 'logs' | 'metrics' | 'config' | 'fetch', '/explorer-configuration'>,
```
Comment (agusaldasoro) [inline src/controllers/handlers/about-handler.ts:22]: is it okay with the `/` prefix? `/explorer-configuration`?

File: src/controllers/handlers/about-handler.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    comms: {
+      healthy: false
+    },
+    lambdas: {
+      healthy: false
+    }
+  }
+
+  try {
+    const response = await fetch.fetch(`${lambdasUrl}/health`, {
```
Comment (agusaldasoro) [inline src/controllers/handlers/about-handler.ts:49]: should we check the response code here too?

File: src/controllers/handlers/about-handler.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      method: 'GET',
+      headers: {
+        Accept: 'application/json'
+      }
+    })
+
+    const data: Record<string, string> = await response.json()
+
+    body.content.healthy = data['content'] === 'Healthy'
```
Comment (agusaldasoro) [inline src/controllers/handlers/about-handler.ts:58]: if `data['content']` is undefined this will be false, right?

**Review by agusaldasoro — APPROVED** [review]

File: src/controllers/handlers/about-handler.ts (hunk 2 above)
Comment (hugoArregui) [reply src/controllers/handlers/about-handler.ts:49]: I don't think so, I added some unit tests

File: src/controllers/handlers/about-handler.ts (hunk 3 above)
Comment (hugoArregui) [reply src/controllers/handlers/about-handler.ts:58]: I don't think so, I added some unit tests

**Pull request merged**